src/Farmer/Builders/Builders.OperationsManagement.fs

```diff
@@ -0,0 +1,120 @@
+[<AutoOpen>]
+module Farmer.Builders.OperationsManagement
+
+open Farmer
+open Farmer.Arm.OperationsManagement
+
+type SolutionProperties =
+    { Workspace: IBuilder option
+      // todo - should these be IBuilders to access ResourceIds?
+      ContainedResources: string list
```

Comment:
I can't quite tell from the docs if these need to be Resource Ids, nor do I feel like I understand enough about this feature to even test it fully.